docs/source/sysadmin_guide/about_dynamic_consensus.rst

```diff
@@ -0,0 +1,247 @@
+.. _about-dynamic-consensus-label:
+
+***********************
+About Dynamic Consensus
+***********************
+
+Sawtooth :term:`dynamic consensus` lets you choose from a variety of consensus
+algorithms for the network, including PBFT, PoET, Raft, and Devmode.
+
+* For a new network, the administrator for the first node specifies the
+  consensus settings in the :term:`genesis block`. All administrators must
+  must install and start a consensus engine (the same version) and any
+  transaction processors required for consensus (such as PoET Validator
+  Registry).
+  See :doc:`/sysadmin_guide/setting_up_sawtooth_poet-sim` for this procedure.
+
+* To change consensus for a running network, administrators must coordinate the
+  changes for all nodes. Each administrator must install and start the new
+  consensus engine. Then one administrator submits a transaction to change the
+  on-chain consensus settings.
+
+  After the consensus has changed, each administrator can stop the previous
+  consensus engine and any unnecessary transaction processors
+  (such as PoET Validator Registry).
+
+The rest of this topic summarizes the settings and requirements for each
+available consensus type: :ref:`PBFT <pbft-consensus-label>`,
+:ref:`PoET <poet-consensus-label>`, :ref:`Raft <raft-consensus-label>`, and
+:ref:`Devmode <devmode-consensus-label>`.
+
+.. note::
+
+   **Default Consensus Settings:**
```

Comment:
I think it would be worth adding something to the beginning of this section that the `sawtooth.consensus.algorithm.name/version` settings are required. for current versions; if you only read this, you might think what's explained below is something you could use, but you can't. Also, "Default Consensus Settings" probably isn't the best term to use, because there's not "default" consensus (as you explain below). "Backwards Compatibility of Consensus Settings" or something similar would be better.